RevenueCatUI/CustomerCenter/Views/CompatibilityTopBarTrailing.swift

```diff
@@ -12,6 +12,7 @@
 //  Created by Josh Holtz on 8/16/24.
 
 #if CUSTOMER_CENTER_ENABLED
+#if os(iOS)
```

Comment:
Should we port this to `main`?

Reply:
yes, they were in #4226 which I just merged to main